packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx

```diff
@@ -46,6 +46,18 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
     Math.abs(Number(packSizeController.selected?.value || 1));
   const { items } = useOutboundRows();
 
+  const onChangePackSize = (newPackSize: number) => {
+    const previousPackSize = packSizeController.selected?.value || 1;
+    const newQuantity =
+      previousPackSize > newPackSize ? quantity * previousPackSize : quantity;
```

Comment:
This `quantity` (assigned out of scope of this context) was troublesome for me reading through again. Maybe we could change it to `allocatedNumPacks` or something similar?